integration_test/testutil_test.go

```diff
@@ -117,30 +117,31 @@ func lines(in []byte) []string {
 	return strings.Split(trimmed, "\n")
 }
 
-func (it *ITest) lookupExecutable(file string) error {
+func (it *ITest) lookupExecutable(file string) (string, error) {
 	orig := os.Getenv("PATH")
 	defer func() { os.Setenv("PATH", orig) }()
 
 	binPath := filepath.Join(it.Root(), "bin")
 	os.Setenv("PATH", binPath)
 
-	_, err := exec.LookPath(file)
-	return err
+	return exec.LookPath(file)
 }
 
-// AssertExecutableInPATH asserts that the executable file is in bin path.
-func (it *ITest) AssertExecutableInPATH(file string) {
+// AssertExecutableInPATH asserts that the executable file is in bin path and returns its location.
+func (it *ITest) AssertExecutableInPATH(file string) string {
```

Comment:
I'm not 100% happy that this function returns something, on the other hand it should be quite clear what it is. WDYT?

Reply:
I think I wouldn't change the signature here.  You can just drop the return value. Assertion methods typically don't return things.

Reply:
I totally agree. I exposed `LookupExecutable` instead.